crmsh/bootstrap.py

```diff
@@ -1049,6 +1049,20 @@ def generate_ssh_key_pair_on_remote(
     )
     if result.returncode != 0:
         raise ValueError(codecs.decode(result.stdout, 'utf-8', 'replace'))
+
+    cmd = '''
+[ -f ~/.ssh/id_rsa.pub ] || ssh-keygen -y -f ~/.ssh/id_rsa > ~/.ssh/id_rsa.pub
+'''
```

Comment:
This can be append to the previous `cmd` and called with one ssh session, making it faster.

Reply:
Changed